worktree_test.go

```diff
@@ -3207,3 +3207,90 @@ func (s *WorktreeSuite) TestRestoreBoth(c *C) {
 		{Worktree: Untracked, Staging: Untracked},
 	})
 }
+
+func TestFilePermissions(t *testing.T) {
+
+	// Initialize an in memory repository
+	remoteUrl := t.TempDir()
+
+	inMemoryFs := memfs.New()
+	remoteFs := osfs.New(remoteUrl)
+	remoteStorage := filesystem.NewStorage(remoteFs, cache.NewObjectLRUDefault())
+
+	remoteRepository, err := Init(remoteStorage, inMemoryFs)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
```suggestion
	assert.NoError(t, err)
```

This will require an import for `"github.com/stretchr/testify/assert"`.